app/controllers/api/servers_controller.rb

```diff
@@ -6,10 +6,11 @@ class ServersController < ApplicationController
 
     skip_before_action :verify_authenticity_token
 
-    before_action :set_server, only: [:show, :update, :destroy, :panic]
+    before_action :verify_lb_checksum
```

Comment:
I think you can drop the method and just do `before_action -> { verify_checksum(true) }`